pkg/apis/core/v1alpha1/kubernetesdiscovery_types.go

```diff
@@ -18,9 +18,138 @@ limitations under the License.
 package v1alpha1
 
 import (
+	"context"
+
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+
+	"github.com/tilt-dev/tilt-apiserver/pkg/server/builder/resource"
+	"github.com/tilt-dev/tilt-apiserver/pkg/server/builder/resource/resourcestrategy"
 )
 
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// KubernetesDiscovery
+// +k8s:openapi-gen=true
+type KubernetesDiscovery struct {
```

Comment:
you also need to add this to register.go i think

Reply:
Ahh good catch - fyi had to update one of the CLI tests to be more lenient (output width/spacing is based on longest resource name, and our new friend `k17y` is annoyingly long) - b0ab895fdb99f8ebb9ad428af18f3fa0d9bc2d20